src/components/home.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable max-len */
+import React from 'react';
+import styles from './home.module.css';
+
+const Home = () => (
+  <>
+    <div className={styles.home_container}>
+      <h1 className={styles.home_title}>Welcome to our page!</h1>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+        <br />
+        <br />
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      </p>
+    </div>
+  </>
+);
+
```

Comment:
Good job working on the project requirements. Firstly it is against Microverse policy to disable eslint or any linters because linters help us write clean code. So it would be nice if you enabled eslint in this file and in other files were it has been disabled.

Secondly it would be nice if you stick to the wireframe design I noticed your page content is too close to the navigation bar it would be nice if you adjusted this and centred it by using margins and padding.

This is your design:

![Screenshot 2023-07-12 at 6 15 50 AM](https://github.com/HFG43/math_magicians/assets/79506105/b3b16b28-e3a9-4d46-a9ed-90b598ea4e76)

Please stick to this wireframe design:
![Screenshot 2023-07-12 at 6 16 30 AM](https://github.com/HFG43/math_magicians/assets/79506105/bc6b620d-044e-41fd-8bd0-207f2ccbdf0c)